### PR TITLE
exception won't cause runtime failure

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

#149 
#146 
#145 

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Instead of propagating exceptions we do nothing

## Description

Change return_exceptions flag to true

### CODE REVIEW

**Comment by harshad16** [comment]
this is issues are still persisting:  https://github.com/thoth-station/package-update-job/blob/8d087a6744f5e8532703920a827cadb60b912e02/producer.py#L156

**Comment by KPostOffice** [comment]
Locally my version is continuing after an `Exception`

**Comment by harshad16** [comment]
> Locally my version is continuing after an `Exception`

can you please elaborate?

**Comment by harshad16** [comment]
the change I m suggesting is to make : 
https://github.com/thoth-station/package-update-job/blob/8d087a6744f5e8532703920a827cadb60b912e02/producer.py#L157
as `warning`.

i dont know what would returning exception on packages not found in specific index, would help us to do.
i have tried the change with https://github.com/harshad16/package-update-job/blob/846c506f7b86c586eead622118fc641256f02b4e/producer.py#L162
results can be seen in : https://console-openshift-console.apps.ocp.prod.psi.redhat.com/k8s/ns/thoth-test-core/pods/package-update-62tcm/logs


package-update-job

**Comment by KPostOffice** [comment]
Ahh, I see. I'll change it to warning

File: producer.py [hunk 1]
```diff
@@ -83,10 +94,11 @@ async def _check_package_availability(
             _LOGGER.info("%r no longer provides %r", package[1], package[0])
             return False
         except Exception as e:
-            _LOGGER.exception("Failed to publish with the following error message: %r", e)
+            _LOGGER.warning("Failed to publish with the following error message: %r", e)
```
Comment (harshad16) [inline producer.py:97]: not sure if we should change all exception to warning, 
these could be helpful to be a exception , as we would know something is wrong while sending message.

WDYT @KPostOffice ?

Comment (harshad16) [reply producer.py:97]: i see some of the message warnings was turned back to exception, can we change this once as well back to the exception.
this is the remaining exception for kafka message.
WDYT  @KPostOffice ?

**Review by harshad16 — APPROVED** [review]
lgtm :+1: 
thanks

**Pull request merged**